cmd/proxy/actions/app_proxy.go

```diff
@@ -133,6 +133,16 @@ func getSingleFlight(c *config.Config, checker storage.Checker) (stash.Wrapper,
 			return nil, fmt.Errorf("Redis config must be present")
 		}
 		return stash.WithRedisLock(c.SingleFlight.Redis.Endpoint, checker)
+	case "redis-sentinel":
+		if c.SingleFlight == nil || c.SingleFlight.Redis == nil {
+			return nil, fmt.Errorf("Redis config must be present")
```

Comment:
Could you mention the names of both the variables in `config.dev.toml` so people know what they need to set? 

(I know we don't get this detailed in many of our error messages, but I'm trying to make things more helpful)